### PR TITLE
Add info on teaching automations

### PR DESCRIPTION
**Pull request by robmarkcole**
**Description:**
Add info on teaching automations, since the teaching events were removed

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/image_processing.facebox.markdown [hunk 1]
```diff
@@ -118,6 +118,56 @@ A valid service data example:
...
+    data_template:
+      message: '{{ trigger.event.data.service_data.name }} taught 
+      with file {{ trigger.event.data.service_data.file_path }}'
+      title: Face taught notification
+```
+{% endraw %}
+
+Any errors on teaching will be reported in the logs. If you enable [system_log](https://www.home-assistant.io/components/system_log/) events:
+
+{% raw %}
```
Comment (MartinHjelmare) [inline source/_components/image_processing.facebox.markdown]: Only templates require the raw tags.

Comment (tjorim) [reply source/_components/image_processing.facebox.markdown]: Could you please expand on that a bit more? When to use, when not etc.
This is pretty unclear in the ['documentation documentation'](https://developers.home-assistant.io/docs/en/documentation_create_page.html) in my opinion.

e.g.

> ## Configuration
> The Configuration Variables section must use the {% raw %}{% configuration %} ... {% endconfiguration %}{% endraw %} tag.

The '... tag', I see multiple tags? Should we use both those tags or is it in this case just poorly rendered and do we only use the {% configuration %}-tag?

> ## Embedding Code
> You can use the default markdown syntax to generate syntax highlighted code. For inline code wrap your code in {% raw %}`{% endraw %}.

Are those `{% raw %}`-tags something we should include? Is this again a case of bad rendering and does the documentation actually just mean to use ` for inline code?

Comment (robmarkcole) [reply source/_components/image_processing.facebox.markdown]: @tjorim my reading of it is that only the contents of the yaml (i.e. configuration) files should be enclosed in the `raw`, so my example service data should NOT be enclosed. Is this correct @MartinHjelmare ?

Comment (MartinHjelmare) [reply source/_components/image_processing.facebox.markdown]: We don't want jekyll to try to process the curly braces in jinja templates. So we use the raw tag to temporarily disable tag processing. Read more here eg:
http://ozzieliu.com/2016/04/26/writing-liquid-template-in-markdown-with-jekyll/

So to answer the question, only literal yaml that contains templates, ie curly braces, needs to be contained within raw tags. But of course there are other cases where we need to disable tag processing. Trying to list that is out of scope for this review.

**Review by MartinHjelmare — APPROVED** [review]

**Comment by frenck** [comment]
Merged a little too quickly.. there was a spelling error.

File: source/_components/image_processing.facebox.markdown [hunk 2]
```diff
@@ -118,6 +118,54 @@ A valid service data example:
...
+{% endraw %}
+
+Any errors on teaching will be reported in the logs. If you enable [system_log](https://www.home-assistant.io/components/system_log/) events:
+
+```yaml
+system_log:
+  fire_event: true
+```
+
+you can create an automation to receive notifications on Facebox errors:
```
Comment (frenck) [inline source/_components/image_processing.facebox.markdown:149]: `you` -> `You`

**Comment by frenck** [comment]
Furthermore, this is not related to a parent PR, so it targets an incorrect branch. It should have been the `current` branch.

**Comment by MartinHjelmare** [comment]
No it is related to a parent PR. It was just not linked.

**Comment by frenck** [comment]
@MartinHjelmare In that case, it should have been linked before merging.

Could @robmarkcole or you add the link please?

**Comment by MartinHjelmare** [comment]
This was a follow up of https://github.com/home-assistant/home-assistant.io/pull/5789.

**Comment by frenck** [comment]
Thanks, @MartinHjelmare! 👍 

Parent PR: home-assistant/home-assistant#15439

**Pull request merged**